creedengojava.properties

```diff
@@ -12,3 +12,9 @@ defaults.mavenArtifactId=creedengo-java-plugin
 2.0.0.date=2024-12-18
 2.0.0.downloadUrl=https://github.com/green-code-initiative/creedengo-java/releases/download/2.0.0/creedengo-java-plugin-2.0.0.jar
 2.0.0.changelogUrl=https://github.com/green-code-initiative/creedengo-java/releases/tag/2.0.0
+
+2.1.0.description=Add rule GCI82 and GCI94; Add support for SonarQube 24.12.0; improve library versions
+2.1.0.sqVersions=[9.9,24.12.0]
```

Comment:
The valid `sqVersions` range ends at 10.7. 
To also support 12.12, you'll add a separate `sqcbVersions` line. 
Take a look [here](https://github.com/SonarSource/sonar-update-center-properties/blob/master/checkstyle.properties)

Reply:
done.